src/core/server/metrics/logging/get_ops_metrics_log.test.ts

```diff
@@ -70,46 +71,49 @@ describe('getEcsOpsMetricsLog', () => {
     const missingMetrics = {
       ...baseMetrics,
       process: {},
+      processes: [],
       os: {},
     } as unknown as OpsMetrics;
     const logMeta = getEcsOpsMetricsLog(missingMetrics);
     expect(logMeta.message).toMatchInlineSnapshot(`""`);
   });
 
   it('provides an ECS-compatible response', () => {
-    const logMeta = getEcsOpsMetricsLog(createBaseOpsMetrics());
-    expect(logMeta).toMatchInlineSnapshot(`
+    const logMeta = getEcsOpsMetricsLog(createMockOpsMetrics(testMetrics));
+    expect(logMeta.meta).toMatchInlineSnapshot(`
```

Comment:
The snapshot was getting too big to easily notice a change, so I've changed this assertion to only test the `meta` field.